### PR TITLE
Declare Linux support via glibc versions

### PR DESCRIPTION
**Pull request by bdice**
Companion pull request for https://github.com/rapidsai/docs/pull/559.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -170,10 +170,11 @@ <h3 class="mt-5 text-white">Requirements</h3>
                     <p><strong>A.</strong> NVIDIA Volta™ or higher GPU with <a
                             href="https://developer.nvidia.com/cuda-gpus" target="_blank">compute capability 7.0+
                             <i class="fa-solid fa-arrow-up-right"></i></a></p>
-                    <p><strong>B.</strong> Ubuntu 20.04 or 22.04, Rocky Linux 8, or WSL2 on Windows 11</p>
+                    <p><strong>B.</strong> Linux or WSL2 on Windows 11</p>
                     <p><strong>C.</strong> Recent <a href="https://docs.nvidia.com/deploy/cuda-compatibility/index.html"
-                            target="_blank"> CUDA version <em>and</em> NVIDIA driver pairs</a>. Check yours with:
+                            target="_blank">CUDA version <em>and</em> NVIDIA driver pairs</a>. Check yours with:
                         <code>nvidia-smi</code>
+                    <p>See <a href="https://docs.rapids.ai/install/#system-req">System Requirements</a> for details</p>
```
Comment (bdice) [inline layouts/index.html:177]:
```suggestion
                    <p>See <a href="https://docs.rapids.ai/install/#system-req">System Requirements</a> for details.</p>
```

File: layouts/index.html [hunk 2]
```diff
@@ -170,10 +170,11 @@ <h3 class="mt-5 text-white">Requirements</h3>
                     <p><strong>A.</strong> NVIDIA Volta™ or higher GPU with <a
                             href="https://developer.nvidia.com/cuda-gpus" target="_blank">compute capability 7.0+
                             <i class="fa-solid fa-arrow-up-right"></i></a></p>
-                    <p><strong>B.</strong> Ubuntu 20.04 or 22.04, Rocky Linux 8, or WSL2 on Windows 11</p>
+                    <p><strong>B.</strong> Linux or WSL2 on Windows 11</p>
```
Comment (raydouglass) [inline layouts/index.html:173]: Maybe say something more like "A compatible Linux distro" and link to the list in the [new docs PR](https://github.com/rapidsai/docs/pull/559/files#diff-bead61f1a2dc63a4fb0ed77ea3c5e653c0625e31588439201f8bdd3c8ca83d67R135)?

**Review by raydouglass — APPROVED** [review]

File: layouts/index.html (hunk 2 above)
Comment (bdice) [reply layouts/index.html:173]: I like this idea! Thank you.

Comment (bdice) [reply layouts/index.html:173]:
```suggestion
                    <p><strong>B.</strong> <a href="https://docs.rapids.ai/install/#system-req" target="_blank">Compatible Linux distribution</a> or <a href="https://docs.rapids.ai/install#wsl2" target="_blank">WSL2 on Windows 11</a></p>
```

**Pull request merged**